product-features/liquidity-providers/how-to.md

```diff
@@ -0,0 +1,223 @@
+---
+description: >-
+  In these two guides, you'll find the required steps to provide liquidity and
+  to withdraw liquidity from ALEX DEX pools.
+---
+
+# How to add/remove liquidity
+
+When **adding liquidity**, you will deposit an equivalent value of both tokens into the pool. In return, you'll receive LP tokens, which represent your share of that specific liquidity pool.
+
+When **removing liquidity**, you will transfer your LP tokens back to the ALEX smart contract and receive your original funds plus any fees accrued while holding those LP tokens.
```

Comment:
When **removing liquidity**, you will transfer your LP tokens back to the ALEX smart contract and withdraw an equivalent value of the token pair plus any fees accrued while holding those LP tokens. The amount of the tokens relative to each other may have changed since they were deposited, which is referred to as "Impermanent loss".

To Sofia:
Suggest having a separate paragraph to explain Impermanent loss, since it is relatively hard to understand.